submissions/okkkko/html_css_popup/style.css

```diff
@@ -0,0 +1,151 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  list-style-type: none;
+  font-family: "Karla", sans-serif;
+}
+header {
+  padding: 15px;
+  min-width: 315px;
+  background-color: #538687bd;
+}
+body {
+  background-color: #e2e0e033;
+}
+a:visited,
+a {
+  text-decoration: none;
+  color: black;
+}
+a {
+  display: block;
+  text-align: center;
+}
+.link_icon {
+  width: 26px;
+  height: 26px;
+  cursor: pointer;
+}
+#images {
+  width: 83px;
+}
+label img {
```

Comment:
would be better to add selectors for tags and replace styles for it. It good practice in case if someone later will use the same tags anywhere on the same page

Reply:
Thank you for review!